drivers/platform/aducm3029/timer_extra.h

```diff
@@ -0,0 +1,68 @@
+/***************************************************************************//**
+ *   @file   timer_extra.h
+ *   @brief  Header file of TIMER driver for ADuCM302x
+ *   @author Mihail Chindris (mihail.chindris@analog.com)
+********************************************************************************
+ * Copyright 2019(c) Analog Devices, Inc.
+ *
+ * All rights reserved.
+ *
+ * Redistribution and use in source and binary forms, with or without
+ * modification, are permitted provided that the following conditions are met:
+ *  - Redistributions of source code must retain the above copyright
+ *    notice, this list of conditions and the following disclaimer.
+ *  - Redistributions in binary form must reproduce the above copyright
+ *    notice, this list of conditions and the following disclaimer in
+ *    the documentation and/or other materials provided with the
+ *    distribution.
+ *  - Neither the name of Analog Devices, Inc. nor the names of its
+ *    contributors may be used to endorse or promote products derived
+ *    from this software without specific prior written permission.
+ *  - The use of this software may or may not infringe the patent rights
+ *    of one or more patent holders.  This license does not release you
+ *    from the requirement that you obtain separate licenses from these
+ *    patent holders to use this software.
+ *  - Use of the software either in source or binary form, must be run
+ *    on or directly connected to an Analog Devices Inc. component.
+ *
+ * THIS SOFTWARE IS PROVIDED BY ANALOG DEVICES "AS IS" AND ANY EXPRESS OR
+ * IMPLIED WARRANTIES, INCLUDING, BUT NOT LIMITED TO, NON-INFRINGEMENT,
+ * MERCHANTABILITY AND FITNESS FOR A PARTICULAR PURPOSE ARE DISCLAIMED.
+ * IN NO EVENT SHALL ANALOG DEVICES BE LIABLE FOR ANY DIRECT, INDIRECT,
+ * INCIDENTAL, SPECIAL, EXEMPLARY, OR CONSEQUENTIAL DAMAGES (INCLUDING, BUT NOT
+ * LIMITED TO, INTELLECTUAL PROPERTY RIGHTS, PROCUREMENT OF SUBSTITUTE GOODS OR
+ * SERVICES; LOSS OF USE, DATA, OR PROFITS; OR BUSINESS INTERRUPTION) HOWEVER
+ * CAUSED AND ON ANY THEORY OF LIABILITY, WHETHER IN CONTRACT, STRICT LIABILITY,
+ * OR TORT (INCLUDING NEGLIGENCE OR OTHERWISE) ARISING IN ANY WAY OUT OF THE USE
+ * OF THIS SOFTWARE, EVEN IF ADVISED OF THE POSSIBILITY OF SUCH DAMAGE.
+*******************************************************************************/
+
+#ifndef TIMER_H
+# define TIMER_H
```

Comment:
nitpick: extra space between `#` and `define`

Reply:
The space is used to indent the defines

Reply:
Your point is valid. But according to all other header files in this repository, the indentation for defines is not applied. This is how some IDEs generate the defines when creating new header file. We should keep the consistency.
https://github.com/analogdevicesinc/no-OS/blob/7c2fb4c532324da088dc6a38cd23d7253f7db645/include/delay.h#L40-L41
https://github.com/analogdevicesinc/no-OS/blob/7c2fb4c532324da088dc6a38cd23d7253f7db645/include/error.h#L40-L41
https://github.com/analogdevicesinc/no-OS/blob/7c2fb4c532324da088dc6a38cd23d7253f7db645/include/fifo.h#L39-L40
https://github.com/analogdevicesinc/no-OS/blob/7c2fb4c532324da088dc6a38cd23d7253f7db645/include/gpio.h#L40-L41
https://github.com/analogdevicesinc/no-OS/blob/7c2fb4c532324da088dc6a38cd23d7253f7db645/include/i2c.h#L40-L41
https://github.com/analogdevicesinc/no-OS/blob/7c2fb4c532324da088dc6a38cd23d7253f7db645/include/irq.h#L40-L41
https://github.com/analogdevicesinc/no-OS/blob/7c2fb4c532324da088dc6a38cd23d7253f7db645/include/spi.h#L40-L41

Reply:
done